flake.nix

```diff
@@ -24,16 +22,25 @@
       # provide a dev shell for all systems that the holonix flake supports
       systems = builtins.attrNames holonix.devShells;
 
-      perSystem = { config, system, pkgs, lib, ... }:
+      perSystem = { inputs', config, system, pkgs, lib, ... }:
         {
           formatter = pkgs.nixpkgs-fmt;
 
           devShells.default = pkgs.mkShell {
-            inputsFrom = [ holonix.devShells.${system}.holochainBinaries ];
-            packages = with pkgs; [
+            inputsFrom = [ inputs'.holonix.devShells ];
+            packages = (with inputs'.holonix.packages; [
+              # add packages from Holonix
+              holochain
+              lair-keystore
+              rust
```

Comment:
I think if you include `devShells`, all packages from holonix are already included https://github.com/holochain/holonix/blob/main/flake.nix#L284-L292. Adding packages separately is redundant.

Reply:
It's only supposed to copy build inputs, not packages -> https://ryantm.github.io/nixpkgs/builders/special/mkshell/#sec-pkgs-mkShell-attributes

We ran into this with the template flake too but I'm wondering why this ever worked... Going to have a look at the previous `holonix` shell and make this line needs to be removed here.

I want it to work in a way where we can list packages because not downloading hc-launch and scaffolding where we don't need them saves quite a bit of data

Reply:
Alright, I see. We should only need holochain, lair-keystore and rust as packages for the Tryorama dev shell. Is there anything else you're looking to include in it?

Reply:
I honestly can't tell from that shells file, there's so much mapping around of packages...

But what this feature should be for is importing build dependencies from a package. So if we import something that needs openssl to build, then using `inputsFrom` with that something would include openssl in the dev shell.

Reply:
> Alright, I see. We should only need holochain, lair-keystore and rust as packages for the Tryorama dev shell. Is there anything else you're looking to include in it?

Nope, just what you listed, I've removed the `inputsFrom` now because you've made me realise it wasn't doing anything :)